setup.py

```diff
@@ -33,12 +49,19 @@
                 'RefRed.thread',
                 ]
 
-setup(name = __package_name__,
+setup(name='RefRed',
       version=versioneer.get_version(),
       cmdclass=versioneer.get_cmdclass(),
-      description = __description__,
-      scripts = __scripts__,
-      author = __author__,
-      author_email = __author_email__,
-      packages = __packages__,
-      url = __url__)
+      description='Liquids Reflectrometer Data Reduction Software',
+      license='GPL version 3.0',
+      scripts=['scripts/RefRed', 'scripts/start_refred.py'],
+      author='Jean Bilheux',
```

Comment:
Is it true?

Reply:
Though he is still the top contributor to this project.

Reply:
You can ask Jean about whether he still wants to be the author. He is listed as the author